manim/mobject/geometry.py

```diff
@@ -402,9 +387,25 @@ def __init__(self, point=ORIGIN, **kwargs):
 
 
 class SmallDot(Dot):
-    CONFIG = {
-        "radius": DEFAULT_SMALL_DOT_RADIUS,
-    }
+    CONFIG = {"radius": DEFAULT_SMALL_DOT_RADIUS}
+
+
+class AnnotationDot(VMobject):
+    def __init__(self, color=BLUE):
+        VMobject.__init__(self)
+        sourunding_dot = Dot().scale(1.3).set_fill(color=WHITE).set_z_index(-1)
```

Comment:
Suggestion: Not really necessary, but maybe we should give the user more control over `surrounding_dot's` color rather than just setting it to white?

Reply:
good idea!